src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = (props) => {
+  const { tabs, onTabSelected, selectedTabId } = props;
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId)
+  || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={cn({ 'is-active': selectedTab === tab })}
+              data-cy="Tab"
+            >
+              <a
+                onClick={() => (selectedTabId === tab.id
+                  ? null
+                  : onTabSelected(tab))}
```

Comment:
Move it into separate function
```suggestion
                onClick={() => (selectedTabId !== tab.id && onTabSelected(tab))}
```